modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -82,7 +82,9 @@ extern "C" {
 
 #include <libavutil/mathematics.h>
 #include <libavutil/opt.h>
+#if LIBAVUTIL_BUILD >= CALC_FFMPEG_VERSION(54, 1, 0)
 #include <libavutil/display.h>
+#endif
```

Comment:
Condition should correlate with `av_display_rotation_get` code below.

relates #21754
/cc @mshabunin 

Correct version conditions are defined here: https://github.com/FFmpeg/FFmpeg/blame/d79c240196f43b93bd204363f1facc270029f113/doc/APIchanges#L1689-L1695

BTW, versions like .100/.101 mush have `&& LIBAVFORMAT_VERSION_MICRO >= 100` check as they are FFmpeg only.

Reply:
Updated versions in the guard macro.